rust/src/structs.rs

```diff
@@ -18,12 +18,12 @@ pub struct TarLine(pub Line<f64>, pub f64);
 impl TarLine {
     /// Create an AABB from the contained `Line`
     pub fn envelope(&self) -> AABB<Point> {
-        let half_d = self.1 / 2.0;
+        let d = self.1;
```

Comment:
I thought single letter variable names were discouraged in Rust?

Reply:
I've never been told that! I feel comfortable with it here though since its not a public method or anything but happy to change